### ISSUE TITLE
Remove dolores-app cf route

### ISSUE THREAD

**Issue by gemfarmer** [18F/dolores-landingham-bot#42]
There should only be one route associated with the `dolores-app` app, `dolores.18f.gov`

**Comment by jessieay** [comment]
Closing in favor of https://trello.com/c/m3AV94On

**Comment by jessieay** [comment]
@gemfarmer is this done?

**Comment by gemfarmer** [comment]
@jessieay this still hasn't been done, but it is **very low** priority. Basically I didn't like that the site exists simultaneously at `dolores.18.gov` and `dolores-app.18f.gov`, but this was done for proxying purposes, and isn't _exactly_ a problem. I think this issue could be nixed in favor of #65, which would change the auth to Mike Bland's [`hmac_authentication` gem](https://rubygems.org/gems/hmac_authentication) and remove the need for proxying. 

tldr; I vote we close this issue and add it the #65.
